rpc_jobs/standard_job_postmerge.yml

```diff
@@ -0,0 +1,140 @@
+- project:
+    name:       "gating-post-merge"
+    repo_url:   "https://github.com/rcbops/rpc-gating"
+    branch:     "master"
+    image:      "xenial"
+    scenario:   "functional"
+    action:     "test"
+    jira_project_key: "RE"
```

Comment:
There's no mention made of this in the docs. I think it should be mentioned and explained.

Reply:
Added todo to the issue to update docs.